tests/unit/test_clidriver.py

```diff
@@ -688,7 +692,38 @@ def test_aws_with_read_timeout(self):
             'lambda invoke --function-name foo out.log --cli-read-timeout 90',
             expected_rc=0)
         call_args = self.create_endpoint.call_args
-        self.assertEqual(call_args[1]['timeout'][1], 90)
+        self.assertEqual(call_args[1]['timeout'], (60, 90))
```

Comment:
I incline to the original line, because it won't couple with a default value of another parameter (i.e. connect_timeout) which is not the test scope of current test case.

Or in case we really want to include the connect_timeout value in this test case, in order to prove this PR's implementation does not incorrectly change the other parameter's default value, I would suggest we refer to that value by importing it from botocore, rather than hardcoding it here.

Same comment applies to the following similar test cases.